projects/power_distribution/src/pd_fan_ctrl.c

```diff
@@ -1,3 +1,5 @@
+#include "pd_fan_ctrl.h"
```

Comment:
Is there a reason this was moved to the top?

Reply:
No I think that was just my auto formatter.